kvui.py

```diff
@@ -703,6 +703,13 @@ def refresh_hints(self, hints):
     def hint_sorter(element: dict) -> str:
         return ""
 
+    def fix_heights(self):
+        """Workaround fix for divergent texture and layout heights"""
+        for element in self.children[0].children:
+            max_height = max(child.texture_size[1] for child in element.children)
+            if element.height != max_height:
+                element.height = max_height
```

Comment:
Could be made unconditional since the end result is that `element.height` is now `max_height`, whether it was before or we are changing it to be so
 
```suggestion
            element.height = max_height
```